guardian/typerighter
# Bump Play to 2.8.11, SBT to 1.5.7

## What does this change?

Bumps Play to 2.8.11, SBT to 1.5.7.

## How to test

Run locally. (Do not deploy to CODE – it's waiting for #168!) The application should work as before.
